packages/crypto/src/primitives/xchacha20-poly1305.ts

```diff
@@ -132,27 +132,25 @@ export class XChaCha20Poly1305 {
    * ```
    *
    * @param params - The parameters for the decryption operation.
-   * @param params.data - The encrypted data to decrypt, represented as a Uint8Array.
+   * @param params.data - The encrypted data to decrypt including the authentication tag,
+   *                      represented as a Uint8Array.
    * @param params.key - The key to use for decryption, represented in JWK format.
    * @param params.nonce - The nonce used during the encryption process.
-   * @param params.tag - The authentication tag generated during encryption.
```

Comment:
tag example needs to be removed from example comment code

Reply:
Good catch.  Thanks!

Addressed in [38ba056](https://github.com/TBD54566975/web5-js/pull/393/commits/38ba056ca3bfc9486f8e1045d245f66f98f31fc0)